Blue:
  LH0_W: -0.013412232750291252
  LH0_X: 0.9173382110531267
  LH0_Y: 0.39786512947201635
  LH0_Z: 0.0037493072315699597
  LH0_x: -1.8365605064706325
  LH0_y: -0.7974890146287529
  LH0_z: -0.004265395087263104
  LH1_W: 0.0
  LH1_X: 0.5
  LH1_Y: 0.0
  LH1_Z: 0.0
  LH1_x: 0.0
  LH1_y: 0.0
  LH1_z: 0.0
  LH2_W: 0.0
  LH2_X: 0.5
  LH2_Y: 0.0
  LH2_Z: 0.0
  LH2_x: 0.0
  LH2_y: 0.0
  LH2_z: 0.0
Yellow:
  LH0_W: 0.0037493072315699597
  LH0_X: -0.39786512947201635
  LH0_Y: 0.9173382110531267
  LH0_Z: 0.013412232750291252
  LH0_x: 1.6736838729684513
  LH0_y: 0.025976779494004898
  LH0_z: 0.005141120000893674
  LH1_W: 0.0
  LH1_X: 0.5
  LH1_Y: 0.0
  LH1_Z: 0.0
  LH1_x: 0.0
  LH1_y: 0.0
  LH1_z: 0.0
  LH2_W: 0.0
  LH2_X: 0.5
  LH2_Y: 0.0
  LH2_Z: 0.0
  LH2_x: 0.0
  LH2_y: 0.0
  LH2_z: 0.0
num_LH: 2
